metrics/deploy/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: ocs-metrics-exporter
+    app.kubernetes.io/name: ocs-metrics-exporter
+    app.kubernetes.io/version: 0.0.1
+  name: ocs-metrics-exporter
+  namespace: openshift-storage
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: ocs-metrics-exporter
+      app.kubernetes.io/name: ocs-metrics-exporter
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: ocs-metrics-exporter
+        app.kubernetes.io/name: ocs-metrics-exporter
+        app.kubernetes.io/version: 0.0.1
```

Comment:
Are we going to be shipping this as a new build in the product? If so, shouldn't this be the same as the current ocs-operator version?

Reply:
No new build is required. Binary is added to ocs-operator image and run as a separate deployment internally.

Reply:
:+1: